langchain/retrievers/multi_query.py

```diff
@@ -0,0 +1,158 @@
+import logging
+from typing import List
+
+from pydantic import BaseModel, Field
+
+from langchain import LLMChain
```

Comment:
we should import from langchain.chains.llm (inside code base, we want to import as direct as possible, in notebooks we can do things like this